docs/UserGuide.md

```diff
@@ -634,7 +634,7 @@ ClientCare shows a success message in the Feedback Display too.
 
 <div style="page-break-after: always;"></div>
 
-### Adding notes to clients: `remark`
+### Adding notes to a client: `remark`
```

Comment:
Would we need to change the following links that refer to this too?

In "client related commands" there is a following link that refers to this:
"* [Adding notes to a client: `remark`](#adding-notes-to-clients-remark)".

In "commands summary" there is also a link that refers to this:
"[**Remark**](#adding-notes-to-clients-remark)".